backend/curation/api/v1/curation/collections/collection_id/versions.py

```diff
@@ -0,0 +1,21 @@
+from flask import jsonify, make_response
+
+from backend.curation.api.v1.curation.collections.common import reshape_for_curation_api
+from backend.layers.common.entities import CollectionId
+from backend.portal.api.providers import get_business_logic
+
+
+def get(collection_id: str):
```

Comment:
we should add a check that the collection_id is a UUID and return an error code (404 or 403, whatever we end up deciding is appropriate) if not.

Reply:
I've seen that you put that in in a few places, though it looks like a mix of 403 and 404.

I'm going to 
1) factor out this code into `common.py`, and 
2) standardize on 403